packages/nimble-components/src/card/tests/card.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from '@microsoft/fast-element';
```

Comment:
Can you also add "matrix" stories for the new component? You can look at some of the existing matrix tests in the repository as an example. The one you create (at least initially) won't need as many test dimensions as most of our existing components, which test for all the various enabled/disabled state, appearances, etc.

`card-button-matrix.stories.ts` might be a good reference since it is somewhat similar to what you'll be adding.

Reply:
Added matrix stories with the theme matrix and the hidden state
![image](https://github.com/ni/nimble/assets/35350751/0013cea9-fc8b-4e48-afa5-a745af164729)
Theme:
![image](https://github.com/ni/nimble/assets/35350751/9322aa9f-5a72-44f0-9b43-2867daf9b121)
Hidden:
![image](https://github.com/ni/nimble/assets/35350751/dcda9183-8772-4c6a-af0c-554730e304b4)